### ISSUE TITLE
Identical condition in bin2llvmir/optimizations/simple_types/simple_types.cpp

### ISSUE THREAD

**Issue by maddin200** [avast-tl/retdec#66]
\src\bin2llvmir\optimizations\simple_types\simple_types.cpp line 485
user->getOpcode() == Instruction::AShr ||

**Comment by mewmew** [comment]
For added context https://github.com/avast-tl/retdec/blob/master/src/bin2llvmir/optimizations/simple_types/simple_types.cpp#L485:

```cpp
			 user->getOpcode() == Instruction::LShr ||
			 user->getOpcode() == Instruction::AShr || // Same as below.
			 user->getOpcode() == Instruction::AShr || // Save as above.
			 user->getOpcode() == Instruction::And ||
```

**Comment by s3rvac** [comment]
Nice catch, thank you! Fixed in ef13f8980034fd94f4f0184107d225326d04b4b4.
